### PR TITLE
Generated Network SDK based on Swagger's October release

### PR DESCRIPTION
**Pull request by anton-evseev**
## Description

- Added operations for ExpressRouteLinks, ExpressRoutePorts, ExpressRoutePortsLocations and LoadBalancerOutboundRules
- Added support for ApplicationGatewayCustomError and ApplicationGatewayFirewallExclusion
- Added support for parameters "MaxRequestBodySizeInKb" and "FileUploadLimitInMb" for ApplicationGatewayWebApplicationFirewallConfiguration
- Added support for "ICMP" value of AzureFirewallNetworkRuleProtocol
- Added support for parameter "AppliedTo" for EvaluatedNetworkSecurityGroup
- Added support for parameters "BandwidthInGbps" and "Stag" for ExpressRouteCircuit
- Added support for "Basic" value of ExpressRouteCircuitSkuTier
- Added support for parameter "VerbosityLevel" for NetworkConfigurationDiagnostic
- Renamed NetworkConfigurationDiagnostic's parameter "Queries" to "Profiles" ([PR](https://github.com/Azure/azure-rest-api-specs/pull/3905), not sure if it's a breaking change or not)

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by number213** [comment]
@MikhailTryakhov @dsgouda  please review if assembly version change is correct

```diff
 [assembly: AssemblyVersion("19.5.0.0")]
-[assembly: AssemblyFileVersion("19.5.1.0")]
+[assembly: AssemblyFileVersion("19.6.0.0")]
```

File: src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
 [assembly: AssemblyVersion("19.5.0.0")]
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs]: Please set `AssemblyVersion` to `19.0.0.0`
`AssemblyFileVersion` is correct

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good apart from a minor comment

File: src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (MikhailTryakhov) [reply src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs]: can we easily downgrade this version?

Comment (dsgouda) [reply src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs]: Correcting myself, please leave `AssemblyVersion` unchanged

Comment (MikhailTryakhov) [reply src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs]: @dsgouda  so can you please merge sdk?

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**